.changes/next-release/feature-encoding-19887.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "feature",
```

Comment:
I still think this would be better to label it as an `enhancement`. Specifically, `feature` typically warrants a larger feature for users to try out and thus a minor version bump of the CLI, but this addition is more to re-enable a work around that we had for v1, but with a different name. For `enhancement`s they only result in patch version bumps.